### ISSUE TITLE
Move to latest Facebook API

### ISSUE THREAD

**Issue by lucasderraugh** [cuappdev/icefishing#67]
Lots have changes have happened since the 4.0 release of the API that we missed when first implementing this support.

**Comment by acheng96** [comment]
The new Facebook API has already been integrated into a separate branch. However, it faces the same issue that we had previously where if you log in once, then it'll always remember you (automatically go to the "You've authorized Facebook" page) when you log out and try logging in as a different user. There doesn't seem to be a good way to fix this issue but once it's been fixed, the branch is ready to be merged into master.
